packages/@aws-cdk/aws-eks/README.md

```diff
@@ -524,8 +556,20 @@ deleted), the next `cdk deploy` will issue a `helm uninstall` command and the
 Helm chart will be deleted.
 
 When there is no `release` defined, the chart will be installed using the `node.uniqueId`,
-which will be lower cassed and truncated to the last 63 characters.
+which will be lower cased and truncated to the last 63 characters.
 
+By default, all Helm charts will be installed concurrently. In some cases, this
+could cause race conditions where two Helm charts attempt to deploy the same
+resource or if Helm charts depend on each other. You can use
```

Comment:
We should be able to compute this automatically? (do we not already in the cdk?) 

Especially when helm charts depend on each other, but also when charts depend on the same resource.

Reply:
We don't "see into" the helm charts, so we can't reason about their contents/dependencies.